kpops/component_handlers/helm_wrapper/helm.py

```diff
@@ -206,8 +212,8 @@ def __execute(self, command: list[str]) -> str:
         log.debug(f"Executing {' '.join(command)}")
         process = subprocess.run(
             command,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
+            check=True,
+            capture_output=True,
```

Comment:
did you verify that those are equal?

Reply:
`check` is an addition, [adds some safety](https://docs.astral.sh/ruff/rules/subprocess-run-without-check/)

`capture_output` is equivalent to the removed lines